docs/actions/bazarliste.yaml

```
@@ -184,6 +193,14 @@ actions:
           '>-1M': _t(AB_bazar_commons2_filter_on_date_for_one_month)
           '>-2Y': _t(AB_bazar_commons2_filter_on_date_for_two_years)
           '>-7D&<+7D': _t(AB_bazar_commons2_filter_on_date_one_week_more_and_less)
+      resetfiltersbutton:
+        label: _t(AB_bazar_commons2_resetfiltersbutton_label)
+        advanced: true
+        type: checkbox
+        default: 1
```

```suggestion
        default: 0
```

Nous avions convenu de ne pas changer l'état actuel de bazarlistedonc ce paramètre DOIT rester à 0 par défaut

---

ok